packages/central-server/src/apiV2/entities/EditEntity.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { BESAdminEditHandler } from '../EditHandler';
+
+export class EditEntity extends BESAdminEditHandler {
+  async updateRecord() {
+    // ensure only name field can be updated
```

Comment:
```suggestion
    // for now, ensure only name field can be updated
    // others may be supported in future but want to avoid anyone editing e.g. code
```